build.gradle

```diff
@@ -416,7 +416,7 @@ distributions {
     contents {
       from("./LICENSE") { into "." }
       from("build/reports/license/license-dependency.html") { into "." }
-      from("./migrations") { into "./migrations" }
+      from("./slashing-protection/src/main/resources/migrations") { into "./migrations" }
```

Comment:
I recall one of this section ends up in the final jar .... and another section is required for "installdist". As installdist uses a different mechanism to place these bits.

Reply:
Do think we need to do this as installDist is intended for developers.